lab_5_scrapper/scrapper.py

```diff
@@ -144,21 +285,35 @@ def parse(self) -> Union[Article, bool, list]:
         """
         Parses each article
         """
-        pass
+        response = make_request(self.full_url, self.config)
+        article = BeautifulSoup(response.text, 'lxml')
+        self._fill_article_with_text(article)
+        self._fill_article_with_meta_information(article)
+        return self.article
 
 
 def prepare_environment(base_path: Union[Path, str]) -> None:
     """
     Creates ASSETS_PATH folder if no created and removes existing folder
     """
-    pass
+    if base_path.exists():
+        shutil.rmtree(base_path)
+    base_path.mkdir(parents=True)
 
 
 def main() -> None:
     """
     Entrypoint for scrapper module
     """
-    pass
+    prepare_environment(ASSETS_PATH)
+    config = Config(path_to_config=CRAWLER_CONFIG_PATH)
+    crawler = Crawler(config=config)
+    crawler.find_articles()
+    for i, full_url in enumerate(crawler.urls, start=1):
+        parser = HTMLParser(full_url=full_url, article_id=i, config=config)
+        article = parser.parse()
+        if isinstance(article, Article):
```

Comment:
why do you use this condition here?

Reply:
code style won't pass if we don't check if article is an Article instance